dev/scripts/advisories_list.sh

```diff
@@ -1,3 +1,5 @@
 #!/bin/bash
 
-curl -v -XGET http://localhost:8080/api/patch/v1/advisories | python -m json.tool
+source "$(dirname $(realpath "$0"))/env.sh"
```

Comment:
Instead of this I would add DEV_MODE env var to manager and when it's active put testing x-rh-identity string using some "middlewares.PutTestingIdentity()" to each request. It enables also calling from browser which is now broken (I guess). Then we can also remove this IDENTITY env var setting and passing to each request.

Reply:
This is how its done in vulnerabilities. I'd like to be able to query the DB as different users, and not introduce different behavior into the base manager code. For unit tests, we use MockAuthenticator.